front/cypress/integration/idea_cards.ts

```diff
@@ -38,31 +38,34 @@ describe('Idea cards without filter sidebar sorting and filtering', () => {
     // sort by newest first
     cy.get('#e2e-ideas-sort-dropdown').click();
     cy.get('.e2e-sort-items').find('.e2e-sort-item-new').click();
-    cy.wait(1000);
 
     // find and check first idea card
     cy.get('#e2e-ideas-list');
-    cy.get('.e2e-idea-card').first().contains(ideaTitle);
+    cy.get('.e2e-idea-card').should((cards) => {
+      expect(cards.first()).to.contain(ideaTitle);
+    });
 
     // sort by most voted
     cy.get('#e2e-ideas-sort-dropdown').click();
     cy.get('.e2e-sort-items').find('.e2e-sort-item-popular').click();
-    cy.wait(1000);
 
     // find and check first idea card
     cy.get('#e2e-ideas-list');
-    cy.get('.e2e-idea-card')
-      .first()
-      .contains('Repellendus reprehenderit quae voluptas quia.');
+    cy.get('.e2e-idea-card').should((cards) => {
+      expect(cards.first()).to.contain(
+        'Repellendus reprehenderit quae voluptas quia.'
```

Comment:
Are we sure this copy is always there? Is it like this in the seed files or something?

Reply:
It comes from this template https://github.com/CitizenLabDotCo/citizenlab/blob/master/back/config/tenant_templates/e2etests_template.yml#L2521, which is targeted specifically to e2e tests. So, maybe it's relatively fine to use the hardcoded value. 
Though I agree it's not perfect, I cannot think of any better solution.